### PR TITLE
#7Task. Dzianis Shupenka

### PR DESCRIPTION
**Pull request by dzianisshupenka**
https://travis-ci.org/github/dzianisshupenka/js-assignments/builds/707912967

### CODE REVIEW

File: task/07-yield-tasks.js [hunk 1]
```diff
@@ -82,7 +116,33 @@ function* getFibonacciSequence() {
...
+                key--;
+                index[key]++
+            }
+        } else {
+            stack.pop();
+            index.pop();
+            key--;
+            index[key]++; 
+        }
+    }
```
Comment (elizavetaShcherbatykhAis) [inline task/07-yield-tasks.js:128]: You can do it much shorter :) please, try to improve your code

**Comment by elizavetaShcherbatykhAis** [comment]
Looks cool! Please check the comments
I'll mark the PR as draft please click "ready for review" when it will be corrected!
Thank you!

**Comment by dzianisshupenka** [comment]
https://travis-ci.org/github/dzianisshupenka/js-assignments/builds/713032874

**Comment by elizavetaShcherbatykhAis** [comment]
Well done! I close PR as accepted

**Pull request closed without merging**